includes/options/class-amp-options-manager.php

```diff
@@ -35,17 +35,19 @@ class AMP_Options_Manager {
 	 * @var array
 	 */
 	protected static $defaults = [
-		'experiences'              => [ self::WEBSITE_EXPERIENCE ],
-		'theme_support'            => AMP_Theme_Support::READER_MODE_SLUG,
-		'supported_post_types'     => [ 'post' ],
-		'analytics'                => [],
-		'auto_accept_sanitization' => true,
-		'all_templates_supported'  => true,
-		'supported_templates'      => [ 'is_singular' ],
-		'enable_response_caching'  => true,
-		'version'                  => AMP__VERSION,
-		'story_templates_version'  => false,
-		'story_export_base_url'    => '',
+		'experiences'                                  => [ self::WEBSITE_EXPERIENCE ],
+		'theme_support'                                => AMP_Theme_Support::READER_MODE_SLUG,
+		'supported_post_types'                         => [ 'post' ],
+		'analytics'                                    => [],
+		'auto_accept_sanitization'                     => true,
+		'all_templates_supported'                      => true,
+		'supported_templates'                          => [ 'is_singular' ],
+		'enable_response_caching'                      => true,
+		'version'                                      => AMP__VERSION,
+		'story_templates_version'                      => false,
+		'story_export_base_url'                        => '',
+		'stories_settings_auto_advance_after'          => '',
+		'stories_settings_auto_advance_after_duration' => 0,
```

Comment:
Separate option entries for every story default is not really scalable. I'd use one entry for all the story default settings. Thoughts?

Also, all the other options here are prefixed `story_`, not `_stories`.

Reply:
@swissspidy Agreed - and I'll also fix the nomenclature.

One issue I ran into is with `register_meta`. The meta field holding all settings needs to be registered in order for the editor to be able to pick it up. However, it requires a `type` value to be specified and the option to specify `array` or `object` theme will be added in WordPress 5.3.

I'm working on adding logic to serialize and deserialize the settings as well as on an updated sanitizer to be able to handle the meta contents.

Reply:
The meta fields can still be individual and not arrays. But in the Options  Manager class an array makes more sense.

Reply:
Serialized meta fields prevent things like easy filtering via `WP_Query` too :/